### DISCUSSION TITLE
关于光标输入

### DISCUSSION THREAD

**Discussion by bleedyao** [xiaweiss/paff#3] [General]
作为编辑器，光标是不是应该始终保持在左上角，而不是现在，光标可以处于组件任意位置。

**Comment by xiaweiss** [comment]
目前按 readme 的列表来推进的，只完成了光标动画。

任意位置是希望点击后，可以使用定位来调整光标位置

**Comment by bleedyao** [comment]
在任意光标位置输出文字，可达成光标输入的任务。

而不是先将光标固定在第一行或第一列，再做文字输入。

是这样的吗？

**Reply by xiaweiss** [reply]
好的，我改下，感谢建议
